### PR TITLE
[AT][LanguageKotlin][main][comments][submit] testAddCommentInformationAfterClickingWriteCommentButton() <NataliiaOliver>

### PR DESCRIPTION
**Pull request by NataliiaOliver**
Created testAddCommentInformationAfterClickingWriteCommentButton(), methods and locators added

[TC] - https://trello.com/c/y292TnXD/853-tclanguagekotlinmaincommentssubmit-verify-that-after-clicking-writecomment-butoon-the-add-comment-heading-informational-text-and
[AT] - https://trello.com/c/IYIk8ph6/855-atlanguagekotlinmaincommentssubmit-testaddcommentinformationafterclickingwritecommentbutton-nataliiaoliver

### CODE REVIEW

**Review by SafronovaOlga — APPROVED** [review]

**Comment by NataliiaOliver** [comment]
After first approval I made new commit “code refactored” where on line 69 changed Assert.assertFolse to Assert.assertNotEguals for easier code reading

**Review by verafes — APPROVED** [review]

File: src/test/java/tests/browse_languages/languages/KotlinLanguageTest.java [hunk 1]
```diff
@@ -45,4 +45,37 @@ public void testKotlinPageContent() {
...
+        openBaseURL()
+                .clickBrowseLanguagesMenu()
+                .clickKSubmenu()
+                .clickKotlinLanguage();
+
+        KotlinLanguagePage kotlinLanguagePage = new KotlinLanguagePage(getDriver());
+
+        String actualOldH2HeaderAddComment = kotlinLanguagePage.getH2HeaderAddCommentText();
+
+        Assert.assertNotEquals(actualOldH2HeaderAddComment, expectedH2HeaderAddComment);
```
Comment (AleksandrAkimovChg) [inline src/test/java/tests/browse_languages/languages/KotlinLanguageTest.java:69]: предлагаю так:
        Assert.assertTrue(actualOldH2HeaderAddComment.isEmpty(), "Add comment is visible");

**Review by AleksandrAkimovChg — APPROVED** [review]

**Comment by NataliiaOliver** [comment]
Good idea! got it! thank you for approves!

File: src/test/java/tests/browse_languages/languages/KotlinLanguageTest.java (hunk 1 above)
Comment (NataliiaOliver) [reply src/test/java/tests/browse_languages/languages/KotlinLanguageTest.java]: Good idea! Got it!

**Pull request merged**